modules/rpc/thrift_test.go

```diff
@@ -27,18 +27,56 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
+	"go.uber.org/fx/config"
 	"go.uber.org/fx/modules"
 	"go.uber.org/fx/service"
 	"go.uber.org/yarpc/api/transport"
 	"go.uber.org/yarpc/encoding/thrift"
+	"gopkg.in/yaml.v2"
 )
 
+type testHost struct {
+	service.Host
+	config config.Provider
+}
+
+func (h testHost) Config() config.Provider {
+	return h.config
+}
+
 func TestThriftModule_OK(t *testing.T) {
 	chip := ThriftModule(okCreate, modules.WithRoles("rescue"))
 	dale := ThriftModule(okCreate, modules.WithRoles("ranges"))
 
-	mci := mch()
-	goofy, err := chip(mch())
+	type Modules struct {
```

Comment:
Nit: prefer a type name that makes it clear it's a test. To avoid shadowing other Modules in this package (I think?)